test/expect-tests/fsevents/fsevents_tests.ml

```diff
@@ -171,6 +171,7 @@ let test_with_multiple_fsevents ~setup ~test:f =
               Fsevents.create ~paths:[ config.dir ] ~latency:0.0
                 ~f:(make_callback sync ~f:config.on_event)
             in
+            Fsevents.set_exclusion_paths res ~paths:config.exclusion_paths;
```

Comment:
Why not pass these paths to `Fsevents.create`? Is it useful to be able to set them separately?

Reply:
Do you mean, why not modify `Fsevents.create` to take the exclusion paths as a parameter? It's because I made the bindings reflect the FSEvents api and it's a separate function there. Usually I try not to deviate from the low level bindings too much as it's an easy way to introduce original bugs :)